src/strands_agents_builder/strands.py

```diff
@@ -42,6 +46,7 @@
     store_in_kb,
     strand,
     welcome,
+    rich_interface,
```

Comment:
`rich_interface` is not part of `strands_tools`, which is auto-loaded when `strands` running in own directory.